.github/workflows/format-prettier.yml

```diff
@@ -0,0 +1,50 @@
+name: Format with Prettier
```

Comment:
Why is this file newly added in diff??? I'm confused.

Reply:
Not sure cuz I pulled from dev too so it came from dev